pandapower/control/controller/pq_control.py

```diff
@@ -0,0 +1,222 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2016-2024 by University of Kassel and Fraunhofer Institute for Energy Economics
+# and Energy System Technology (IEE), Kassel. All rights reserved.
+
+from pandapower.control.controller.const_control import ConstControl
+try:
+    from pandaplan.core import pplog
+except:
+    import logging as pplog
+import numpy as np
+
+logger = pplog.getLogger(__name__)
+
+
+class PQController(ConstControl):
```

Comment:
is there a good reason to inherit from const_control?

Reply:
yes, none of the functions defined in ConstControl are redefined/overwritten so without, the code wouldn't work.